opennms-provision/opennms-provisiond/src/main/java/org/opennms/netmgt/provision/service/CoreImportActivities.java

```diff
@@ -148,9 +148,11 @@ public static void scanNode(final ImportOperation operation, final RequisitionIm
         }
         Objects.requireNonNull(monitor);
 
-        if (rescanExisting == null || Boolean.valueOf(rescanExisting) ||
+        // Align with NMS-13040, dbonly will only persist node without scanning
+        if (!"dbonly".equalsIgnoreCase(rescanExisting)
```

Comment:
This breaks https://issues.opennms.org/browse/NMS-12974.

First insertion will always scan ignoring `rescanExisting` flag.  
It doesn't make sense to check `dbonly` for nodes that are getting inserted  for the first time.

Reply:
I am also very confuse with this part. It seems to me the dbonly will same as rescan-existing = false in this case.